tools/ubuntu-setup/scala.sh

```diff
@@ -2,7 +2,8 @@
 set -e
 set -x
 
-wget www.scala-lang.org/files/archive/scala-2.11.6.deb -O /tmp/scala-2.11.6.deb
-sudo dpkg -i /tmp/scala-2.11.6.deb
+#wget www.scala-lang.org/files/archive/scala-2.11.6.deb -O /tmp/scala-2.11.6.deb
+wget https://downloads.lightbend.com/scala/2.11.11/scala-2.11.11.deb -O /tmp/scala-2.11.11.deb
```

Comment:
Use `2.11.8`

Reply:
Why? We're effectively on 2.11.11.

@bwmcadams Can you try to throw out Scala compiler installation? Gradle should handle that for us anyway.

Reply:
Yup. Note this includes the REPL as well, but I don't think people should be doing REPL debugging on their host machine(s) anyway.